Rover/ekf_check.cpp

```diff
@@ -112,11 +112,7 @@ bool Rover::ekf_over_threshold()
         return true;
     }
 
-    if (ekf_position_ok()) {
```

Comment:
I said I wouldn't comment on more instances of these - but I *really* don't like this one.

Reply:
I do like it :P Becomes a style thing, though...